.github/workflows/tests.yml

```
@@ -14,11 +14,10 @@ jobs:
     strategy:
       matrix:
         # nightly-latest to catch anything that breaks these tests in current development
-        # 2/18/2023 version is the first that supports logging
+        # 2/18/2023 version is the first that supports logging, but it is not supportied by dafny-setup-action 1.6.1
```

```suggestion
        # 2/18/2023 version is the first that supports logging, but it is not supported by dafny-setup-action 1.6.1
```
Typo, though it's not really important.